linkml_runtime/utils/schemaview.py

```diff
@@ -1323,6 +1323,10 @@ def get_classes_by_slot(self, slot: SlotDefinition, include_induced: bool = Fals
 
         return list(dict.fromkeys(slots_list))
 
+    @lru_cache()
+    def get_slots_by_enum(self, enum_name: ENUM_NAME = None) -> List[SlotDefinition]:
+        return [s for s in self.all_slots().values() if s.range == enum_name]
```

Comment:
Everything looks perfect, great job @amc-corey-cox. Would you mind adding a quick docstring to this method? I'm approving the PR, so feel free to merge it in once you've added the docstring.

Reply:
Of course. I just added the docstring. Thanks for the review!